test/e2e.v3.test.ts

```diff
@@ -119,25 +119,23 @@ describe("v3 E2E Test Scenarios", () => {
       // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
       // @ts-ignore
       expect(wrappedDocument.key1).toEqual(expect.stringContaining("test"));
-      expect(wrappedDocument.proof.signature.type).toBe("SHA3MerkleProof");
-      expect(wrappedDocument.proof.signature.targetHash).toBeDefined();
-      expect(wrappedDocument.proof.signature.merkleRoot).toBeDefined();
-      expect(wrappedDocument.proof.signature.proof).toEqual([]);
-      expect(wrappedDocument.proof.signature.merkleRoot).toBe(wrappedDocument.proof.signature.targetHash);
+      expect(wrappedDocument.proof.type).toBe("OpenAttestationMerkleProofSignature2018");
+      expect(wrappedDocument.proof.targetHash).toBeDefined();
+      expect(wrappedDocument.proof.merkleRoot).toBeDefined();
```

Comment:
should this be proof.proofs.merkleRoot?

Reply:
no proofs ar ejust hash proofs for the merkle tree